actions/lib/action.py

```diff
@@ -15,6 +15,9 @@ def digitalocean_obj_to_dict(obj):
         # digitalocean.baseapi.BaseAPI has a field named _log that doesn't like to be serialized
         if "_log" in obj_dict:
             del obj_dict['_log']
+        # digitalocean.baseapi.BaseAPI has a field named _log that doesn't like to be serialized
```

Comment:
should this commet be _session instead of _log... also might be cleaner to do this in a loop over an array of fields to delete?

Reply:
It should be, but I don't think so. If it becomes three or more fields, then yes we should do it that way. I'll fix this in a sec.